### ISSUE TITLE
ERROR: Patch application has failed

### ISSUE THREAD

**Issue by BakiDance** [Frogging-Family/wine-tkg-git#695]
I'm running a fresh install of Pop!_OS 21.10. I set `_nomakepkg_dep_resolution_distro="debuntu"` as described on the FAQ page, but proton fails to compile regardless. I may be missing dependencies based on the terminal output, but those aren't available in my repos. I'm unsure how to proceed. I tried installing the prebuilt releases, and while it does show up in Steam, it fails to launch any games.  (I saw that this may be related to a glibc version older than 2.31, but my version is 2.34)

[prepare.log](https://github.com/Frogging-Family/wine-tkg-git/files/8084632/prepare.log)
[TerminalOutput.txt](https://github.com/Frogging-Family/wine-tkg-git/files/8084951/TerminalOutput.txt)

**Comment by Tk-Glitch** [comment]
You are missing the autoconf package.

**Comment by BakiDance** [comment]
Thank you! I installed the package and I did get further this time around, but ultimately it still failed.

[TerminalOutput.txt](https://github.com/Frogging-Family/wine-tkg-git/files/8130637/TerminalOutput.txt)
[prepare.log](https://github.com/Frogging-Family/wine-tkg-git/files/8130639/prepare.log)

**Comment by Tk-Glitch** [comment]
You probably tried at a time where some changes just landed upstream and I needed to rebase stuff. It should work currently.

However, as I noticed you're using Pop, it's unlikely you'll be able to build the resulting source directly - unless Pop!_OS broken packages have been fixed since the last tests, which didn't seem likely to happen ever.
See https://github.com/Tk-Glitch/PKGBUILDS/wiki/wine-tkg-git#dependencies

Using a VM or a container with say Ubuntu might be needed to get things done unless you're open to moving to a different distro.
